main.lua

```diff
@@ -154,7 +166,7 @@ function mod:shouldDeHook()
 		Game():GetLevel():GetStage() > LevelStage.STAGE7,
 		not self.initialized,
 		Game():GetLevel():IsAscent(),
-		Game().Challenge > 0,
+		not self.storage.gameHasTreasure,
```

Comment:
This will throw an error on continuing a run after updating. As self.storage.gameHasTreasure is not initialised.

Reply:
This won't throw an error since it checks if its nil anyways. It just won't show a planetarium chance for continued runs from before the update, which is fine since its temporary.